src/serlo-editor/plugins/rows/editor-renderer.tsx

```diff
@@ -192,17 +177,67 @@ export function EditorRowRenderer({
   return (
     <>
       {draggingAbove ? dropPreview : null}
-      <div ref={container}>
+      <div
+        ref={container}
+        className={clsx(
+          'rows-editor-renderer-container',
+          'border-l-2 border-transparent transition-colors',
+          tw`
+          focus-within:border-gray-600
+          hover:!border-gray-300
+          hover:focus-within:!border-gray-600
+          [&:has(.rows-editor-renderer-container:focus-within)]:border-transparent
+          [&:hover:has(.rows-editor-renderer-container:focus-within)]:!border-gray-300
+          `,
+          tw`
+          [&:focus-within>.rows-tools]:opacity-100
+          [&:has(.rows-editor-renderer-container:focus-within)>.rows-tools]:opacity-0
+          [&:hover>.rows-tools]:!opacity-100
+          `
+        )}
+      >
+        {renderDragButton()}
         <div
           className={collectedDragProps.isDragging ? 'opacity-30' : undefined}
         >
-          {row.render(pluginProps)}
+          {row.render()}
         </div>
       </div>
       {!draggingAbove ? dropPreview : null}
     </>
   )
 
+  function renderDragButton() {
```

Comment:
Would you maybe consider extracting this into a separate component?

Reply:
good idea